dockerfiles/app/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG BASE_IMAGE
+ARG IMAGE_TAG=latest
+FROM ${BASE_IMAGE}:${IMAGE_TAG}
+
+WORKDIR /opt/app
+
+COPY . .
+
+RUN source scl_source enable devtoolset-8 && \
+    npm install
+
+EXPOSE 3001
+
+RUN bash core-js-fix.sh
```

Comment:
Is this still needed - I thought we removed it, no?

Reply:
It was clawed back due to web-app compatibility issues.